test/unit/colorspace_spec.js

```diff
@@ -0,0 +1,563 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Dict,
+  isRef,
+  Name,
+  Ref
+} from '../../src/core/primitives';
+import { Stream, StringStream } from '../../src/core/stream';
+import {
+  ColorSpace
+} from '../../src/core/colorspace';
+
+describe('colorspace', function () {
+  class XRefMock {
+    constructor(array) {
+      this.map = Object.create(null);
+      for (let elem in array) {
+        let obj = array[elem];
+        let ref = obj.ref,
+          data = obj.data;
```

Comment:
Single lines should only contain one statement, so if you put this on a new line, make it `let data = obj.data;`.